src/core.h

```diff
@@ -75,7 +75,7 @@ namespace Faunus {
      * keys as the object should be zero after being processed by
      * i.e. `from_json` or similar.
      */
-    struct SingleUseJSON : private json {
+    struct SingleUseJSON : public json {
```

Comment:
This is potentially dangerous as you could access without deleting the entry which is promised by the class

Reply:
I am aware of that. I believe a stricter validation of json will be introduced anyway with the builder and creator patterns.